### ISSUE TITLE
Package does not build on latest flutter (3.22.0) due to issue with an underlying dependency

### ISSUE THREAD

**Issue by SlimShadyIAm** [cph-cachet/research.package#115]
The research package depends on `simple_html_css` which needs to be updated due to changes in the Material package in the the latest version of Flutter.

Someone has made a PR for this: https://github.com/ali-thowfeek/simple_html_css_flutter/pull/32

Waiting for this to get merged. Until then, you need to downgrade Flutter to v3.19.6 to be able to build the app.

**Comment by albermoon** [comment]
Yes, I had the same problem and finally did the same as you. I deleted the package and the widgets which used the package because they were unused and it didn't make sense to keep them in the project
